SimCore/src/SimCore/BiasOperators/PhotoNuclear.cxx

```diff
@@ -61,8 +69,9 @@ G4VBiasingOperation* PhotoNuclear::ProposeOccurenceBiasingOperation(
     double emXsecBiased = std::max(
         emXsecUnbiased + pnXsecUnbiased_ - pnXsecBiased_, pnXsecUnbiased_);
     if (emXsecBiased == pnXsecUnbiased_) {
-      G4cout << "[ PhotoNuclearXsecBiasingOperator ]: [ WARNING ]: "
-             << "Biasing factor is too large." << std::endl;
+      ldmx_log(warn) << 
```

Comment:
This I may reduce to `debug`, if we are too annoyed having this O(20) times per event